packages/world/test/Factories.t.sol

```diff
@@ -134,6 +137,10 @@ contract FactoriesTest is Test, GasReporter {
   }
 
   function testWorldFactoryGas() public {
-    testWorldFactory(address(this));
+    testWorldFactory(address(this), 0);
+  }
+
+  function testFuzzWorldDeploy(address account, uint _salt) public {
```

Comment:
Added fuzzing test for salt.

Reply:
You actually don't need a separate `testFuzzWorldDeploy` test, because `testWorldFactory` takes the `address account` and `uint salt` parameters, Foundry will automatically run it in Fuzz.

Reply:
You actually don't need a separate `testFuzzWorldDeploy` test, Foundry will automatically run `testWorldFactory` as a property-based test because it has parameters.

So we only need `testWorldFactory(address account, uint256 salt)` and `testWorldFactoryGas()`

![image](https://github.com/latticexyz/mud/assets/29184158/1ed97703-b0c3-4301-990a-396e284bc872)